test/cql-pytest/test_tablets.py

```diff
@@ -45,3 +45,10 @@ def test_create_loop_with_tablets(cql, test_keyspace_128_tablets):
     for i in range(100):
         cql.execute(f"CREATE TABLE {table} (p int PRIMARY KEY, v int)")
         cql.execute("DROP TABLE " + table)
+
+
+def test_alter_cannot_change_vnodes_to_tablets(cql, skip_without_tablets):
+    ksdef = "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : '1' } AND TABLETS = { 'enabled' : false }"
+    with new_test_keyspace(cql, ksdef) as keyspace:
+        with pytest.raises(InvalidRequest):
```

Comment:
please add a few select words in a "match=" (no need for the entire error message!) just to prove the failure was really that you can't ALTER - and not something stupid like a syntax error or tablets not even supported.

Reply:
Yup, will do